libraries/apollo-server/test.sh

```diff
@@ -9,7 +9,7 @@ sleep 10
 
 actual="$(curl 'http://localhost:4000/' -H 'Accept-Encoding: gzip, deflate, br' -H 'Content-Type: application/json' -H 'Accept: application/json' -H 'Connection: keep-alive' -H 'DNT: 1' -H 'Origin: http://localhost:4000' --data-binary '{"query":"query {\n  test {\n    result\n  }\n}"}' --compressed)"
 
-expected='{"data":{"test":{"result":"[{\"id\":7,\"email\":\"john@example.com\",\"name\":\"John Doe\"}]"}}}'
+expected='{"data":{"test":{"result":"[{\"id\":1,\"email\":\"john@example.com\",\"name\":\"John Doe\"}]"}}}'
```

Comment:
seems the database previously was in a strange state that cause the ID to always be 7? no idea really.